recipes/r-comorbidity/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = '1.0.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-comorbidity
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/comorbidity_{{ version }}.tar.gz
+  sha256: 243d8c12901860ed5ccd5f7ec633467aff21d25164225ef8dab5ff9f06b41dd6
+
+build:
+  merge_build_host: True 
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip 
+
+  host:
+    - r-base
+    - r-checkmate
+    - r-data.table
+    - r-stringi
+
+  run:
+    - r-base
+    - r-checkmate
+    - r-data.table
+    - r-stringi
+
+test:
+  commands:
+    - $R -e "library('comorbidity')"           # [not win]
+    - "\"%R%\" -e \"library('comorbidity')\""  # [win]
+
+about:
+  home: https://ellessenne.github.io/comorbidity/
+  license: GPL-3.0-or-later
+  summary: Computing comorbidity indices and scores such as the weighted Charlson score (Charlson,
+    1987 <doi:10.1016/0021-9681(87)90171-8>) and the Elixhauser comorbidity score (Elixhauser,
+    1998 <doi:10.1097/00005650-199801000-00004>) using ICD-9-CM or ICD-10 codes (Quan,
+    2005 <doi:10.1097/01.mlr.0000182534.19832.83>). Australian and Swedish modifications
+    of the Charlson Comorbidity Index are available as well (Sundararajan, 2004 <doi:10.1016/j.jclinepi.2004.03.012>
+    and Ludvigsson, 2021 <doi:10.2147/CLEP.S282475>).
+  license_family: GPL3
+  license_file: LICENSE.txt
+extra:
+  recipe-maintainers:
+    - abalter
```

R packages should include `conda-forge/r` as maintainer:

```suggestion
    - conda-forge/r
    - abalter
```